ci-operator/config/openshift/openshift-network-operator/master.yaml

```diff
@@ -0,0 +1,37 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+binary_build_commands: hack/build-go.sh
+canonical_go_repository: github.com/openshift/openshift-network-operator
```

Comment:
Naming wise this should be cluster-network-operator - I'd prefer if we rename the repos now to be consistent with the other operators and not diverge from the start.